packages/atomic/src/components/search/tabs/atomic-tab-section/atomic-tab-section.tsx

```diff
@@ -0,0 +1,185 @@
+import {
+  loadBreadcrumbActions,
+  loadQueryActions,
+  loadSearchActions,
+  loadSearchAnalyticsActions,
+  loadTabSetActions,
+} from '@coveo/headless';
+import {
+  Component,
+  h,
+  Element,
+  State,
+  Listen,
+  Prop,
+  EventEmitter,
+  Event,
+} from '@stencil/core';
+import ArrowDown from '../../../../images/arrow-down.svg';
+import {InitializeBindings} from '../../../../utils/initialization-utils';
+import {getActiveTab} from '../../../../utils/tab-utils';
+import {Button} from '../../../common/button';
+import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-tab-section',
+  styleUrl: 'atomic-tab-section.pcss',
+  shadow: true,
+})
+export class AtomicTabSection {
+  @InitializeBindings() public bindings!: Bindings;
+  @Element()
+  private host!: HTMLElement;
+
+  /**
+   * When provided, this tab will be selected by default when the component loads. Otherwise, the first tab is selected automatically.
+   */
+  @Prop() defaultActiveTab?: string;
```

Comment:
Could we do without this ? Just always have the first tab be selected by default ? It is really necessary to have multiple mode to handle which one is the default tab ?

Reply:
I think there are some scenarios where being able to set a different initial tab, but I'm fine removing this attribute/logic since it is possible to replicate the same behaviour using the tab param in the url... It is just less obvious